rootfs/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/deis/base:0.3.1
```

Comment:
All other tags (for the base image) don't include the 'v', is this now adopted platform-wide?